basic/x16additions.s

```diff
@@ -954,6 +954,120 @@ cedit:
 	bannex_call bannex_x16edit
 	rts
 
+;******************************************************************
+;
+; LIST [ xxxx - xxxx ]
+; List BASIC program currently in memory and pause after first page
+; Spacebar will pause/unpause
+; In paused mode below will work:
+; PageDown will show one page at a time
+; Arrow down will show one line at a time
+;
+;******************************************************************
+nlines	= $0387			; These variables are in KERNAL space
+llen	= $0386			; Could not figure out how to import
+curs_y	= $0383
```

Comment:
rather than numeric constants, please use `.import`.  However, these symbols may not yet be inside of BANNEX.  Please check the Makefile lines 431 and 432.  Line 431 has ZP labels to be imported, and line 432 has absolute labels.  You will need to add the symbols there.

Any labels you added in `declare.s` will also need to be added to the findsym script in order for them to be available in BANNEX.